### ISSUE TITLE
Access violation on Windows

### ISSUE THREAD

**Issue by dsherret** [dprint/dprint#227]
Reproduction steps on Windows:

```
git clone https://github.com/dprint/dprint.git -b seg_fault
cd dprint/crates/dprint
cargo run --release -- check test.ts
```

Runs fine on linux.

Rustc: 1.43.1 (also occurs on nightly)

**Crash Dump**

I have crash dumps of this and it errors within a simple `format!("");` macro after calling into the web assembly code.

![image](https://user-images.githubusercontent.com/1609021/83190222-14d81980-a100-11ea-9667-7aeab50127d5.png)

I can do minor changes to the app or input to get it to not crash... so it only occurs sometimes.

**What I tried**

Tried the following on linux with no issues (also tried `address` sanitzer and compiling in `--release`) -- https://github.com/japaric/rust-san:

```
RUSTFLAGS="-Z sanitizer=memory" cargo +nightly run -Z build-std --target x86_64-unknown-linux-gnu -- check test.ts
```

No problems found.

I've also tried Application Verifier, but it told me the same information that the debugger does.

**Comment by dsherret** [comment]
Tried Dr. Memory. Opened https://github.com/DynamoRIO/drmemory/issues/2299

**Comment by dsherret** [comment]
Resolved by upgrading to Wasmer 1.0 in #318
